### DISCUSSION TITLE
When Mojo will be open-sourced?

### DISCUSSION THREAD

**Discussion by david-ragazzi** [modular/modular#1628] [Q&A]
I've seen on many forums that some people resist adopting Mojo just because it's not open-source yet. I've read comments on YouTube and Reddit like "when it becomes open-source, I'll be interested". I had read in somewhere that Modular's plan were to open source by the end of 2023, but nothing concrete so far. Is there an internal deadline for this?

**Comment by Eprahim-taha** [comment]
Modular Company announced that Mojo 🔥, will be open source in the first quarter of this year. Two and a half months remain until the end of the first quarter. Perhaps Mojo will become open source during the remaining two and a half months.

**Reply by melodyogonna** [reply]
The standard library will be open-sourced in the first quarter. The Mojo compiler itself will still be under closed development for the foreseeable future. If I have to guess I'll say the compiler will be open-sourced some time next year.

**Reply by Eprahim-taha** [reply]
Why doesn't Mojo 🔥 work on replit.com What do you think 🤔?

**Comment by Eprahim-taha** [comment]
1. I'm excited about Mojo 🔥 being open source, when Mojo 🔥 goes open source it will run on replit.com and Google colab

**Comment by Eprahim-taha** [comment]
Take a look at #1627
